orangecontrib/bio/genialis/genapi.py

```diff
@@ -0,0 +1,190 @@
+from genesis import Genesis
```

Comment:
Rearrange imports, read [PEP8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports). Run pep8 command on your code and fix warnings. Also, I suggest to run pylint and fix warnings. Here is one example of well defined pylint settings: [.pylintrc](https://github.com/genialis/resolwe-bio-py/blob/master/.pylintrc).